scripts/docker/kernel.Dockerfile

```diff
@@ -0,0 +1,19 @@
+# syntax=docker/dockerfile:1
+
+FROM centos:centos7.8.2003
+RUN set -eux \
+	&& yum install -y \
+        epel-release && \
+    yum install -y \
```

Comment:
redundant line

Reply:
As discussed, we need to run two `yum installs` because `epel-release` needs to be installed before any packages from its repo can be installed (I believe just `cmake3` in this case).